src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -122,13 +122,13 @@ public function getFormErrorsForJS(FormInterface $form)
                     $error->getMessageTemplate(),
                     $error->getMessagePluralization(),
                     $error->getMessageParameters(),
-                    'form_error'
+                    'validators'
                 );
             } else {
                 $errors[$formId][] = $translator->trans(
                     $error->getMessageTemplate(),
                     $error->getMessageParameters(),
-                    'form_error'
+                    'validators'
```

Comment:
This kinda looks like a bc kreak for me. As this function can break user applications if they use this on their own. (it is on a abstract class)

Also, did you check all the usages if this change is not breaking anything ? 


![image](https://user-images.githubusercontent.com/1446265/185877745-037e1451-257b-4834-a851-4ca888ca20d1.png)

Reply:
I'm not sure about the BC break and all usages. I'll check in details. But is there anyway to check that easily ? Should I find for all its usages the related feature on BO/FO and test it manually ? :/

Reply:
It seems this is not in our API promise https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md